datacatalog/cloud-client/src/main/java/com/example/datacatalog/CreateFilesetEntry.java

```diff
@@ -0,0 +1,119 @@
+                      .build();
+
+      // Use the client to send the API request.
+      EntryGroup entryGroupResponse = dataCatalogClient.createEntryGroup(entryGroupRequest);
+      System.out.printf("\nEntry Group created with name: %s\n", entryGroupResponse.getName());
+
+      // 2. Create a Fileset Entry.
+      // -------------------------------
+      // Construct the Entry for the Entry request.
+      Entry entry =
+              Entry.newBuilder()
+                      .setDisplayName("My Fileset")
+                      .setDescription("This fileset consists of ....")
+                      .setGcsFilesetSpec(
+                              GcsFilesetSpec.newBuilder().addFilePatterns("gs://my_bucket/*")
+                                      .build())
+                      .setSchema(
+                              Schema.newBuilder()
+                                      .addColumns(
+                                              ColumnSchema.newBuilder()
+                                                      .setColumn("first_column")
+                                                      .setType("STRING")
+                                                      .setDescription(
+                                                              "This columns consists of ....")
+                                                      .build())
+                                      .addColumns(
+                                              ColumnSchema.newBuilder()
+                                                      .setColumn("second_column")
+                                                      .setType("STRING")
+                                                      .setDescription(
+                                                              "This columns consists of ....")
+                                                      .build())
+                                      .build())
+                      .setType(EntryType.FILESET)
+                      .build();
+
+      // Construct the Entry request to be sent by the client.
+      CreateEntryRequest entryRequest =
+              CreateEntryRequest.newBuilder()
+                      .setParent(entryGroupResponse.getName())
+                      .setEntryId(entryId)
+                      .setEntry(entry)
+                      .build();
+
+      // Use the client to send the API request.
+      Entry entryResponse = dataCatalogClient.createEntry(entryRequest);
+      System.out.printf("\nEntry created with name: %s\n", entryResponse.getName());
+    } catch (AlreadyExistsException | IOException e) {
+      // AlreadyExistsException's are thrown if EntryGroup or Entry already exists.
+      // IOException's are thrown when unable to create the DataCatalogClient,
+      // for example an invalid Service Account path.
+      System.out.println("Error in create entry process:\n" + e.toString());
+    }
+  }
+}
+// [END datacatalog_create_fileset_tag]
```

Comment:
```suggestion
// [END datacatalog_create_fileset]
```

Reply:
The tech writer asked to put _tag at end, because he's using it already that way in the docs... is it ok?

Reply:
The tech writer asked to put _tag at end, because he's using it already that way in the docs... is it ok?

Reply:
Sure